cogs/dad.py

```diff
@@ -0,0 +1,34 @@
+import discord
+from discord.ext import commands
+from random import random
+import re
+
+class Dad(commands.Cog, name="dad", command_attrs=dict(hidden=True)):
+    
+    def __init__(self, bot):
+        self.bot = bot
+        self.settings = bot.database_data["dad"]
```

Comment:
Changes:
- Change the `self.settings = bot.database_data["dad"]` in `dad.py` to `self.settings = bot.bot_data["dad"]`
- Add the `"dad": {"probability": 0.1}` in the `config/bot.json` or `config/?.json`